Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cyberark/ubuntu-ruby-fips:latest
+FROM registry.tld/cyberark/ubuntu-ruby-fips:20.04-2.0.7-14
```

Comment:
I assume this won't be version pinned was the base image PR is merged?

Reply:
Correct, pinned just for pipeline verification.

Reply:
Removed. Build will fail until merging https://github.com/cyberark/conjur-base-image/pull/122